src/components/subscriptions/licenses/LicenseManagementTable/LicenseManagementTableBulkActions.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import {
+  ActionRow, Button,
+} from '@edx/paragon';
+import {
+  Email,
+  RemoveCircle,
+} from '@edx/paragon/icons';
+
+import { canRemindLicense, canRevokeLicense } from '../../data/utils';
+
+const LicenseManagementTableBulkActions = ({
+  selectedUsers,
+  bulkRemindOnClick,
+  bulkRevokeOnClick,
+}) => {
+  // Divides selectedUsers users into two arrays
+  const [usersToRemind, usersToRevoke] = useMemo(() => {
+    const tempRemind = [];
+    const tempRevoke = [];
+
+    selectedUsers.forEach(user => {
+      const userStatus = user.status;
+      if (canRemindLicense(userStatus)) {
+        tempRemind.push(user);
+      }
+      if (canRevokeLicense(userStatus)) {
+        tempRevoke.push(user);
+      }
+    });
```

Comment:
nit: Might be a bit cleaner to use `.filter` here? E.g. 

```
const tempRemind = selectedUsers.filter(user => canRemindLicense(user.status));
const tempRevoke = selectedUsers.filter(user => canRevokeLicense(user.status));
```

Reply:
As long as we are ok iterating through the list `2n` instead of `n` times

Reply:
SMH. Good call out, let's keep as is :)